### PR TITLE
refinedet

### PR DESCRIPTION
**Pull request by wuzuowuyou**
refinedet

### CODE REVIEW

File: refinedet/gen_wts_refinedet.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (wang-xinyu) [inline refinedet/gen_wts_refinedet.py:46]: why so many empty lines?

File: refinedet/refinedet.cpp [hunk 2]
```diff
@@ -0,0 +1,1023 @@
...
+#include "torch/torch.h"
+#include "torch/jit.h"
+
+using namespace nvinfer1;
+static Logger gLogger;
+
+//对矩形区域进行修正，防止图片越界
+void RoiCorrect(const cv::Mat &m, cv::Rect &r)
+{
+    if (r.x < 0) r.x = 0;
```
Comment (wang-xinyu) [inline refinedet/refinedet.cpp:24]: pls do not use Chinese in code.  pls check all.

File: refinedet/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+#define USE_FP32  // set USE_INT8 or USE_FP16 or USE_FP32
+
+const int num_class = 25; //包括背景类
+
+//SERIALIZE 表明序列化生成engin，需要设定后面的wts路径（path_wts）和保存engine路径（path_save_engine）
+//INFER 表明是推理模式，需要指定engine路径（path_engine）
+#define INFER    //SERIALIZE   INFER
+
+
```
Comment (wang-xinyu) [inline refinedet/README.md:22]: why you paste configure.h in readme?

**Review by wang-xinyu — CHANGES_REQUESTED** [review]

**Comment by wuzuowuyou** [comment]
Re submit

**Comment by wuzuowuyou** [comment]
1

**Pull request merged**